CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Documentation for hipSPARSE is available at
 ### Changed
 
 * Switch to defaulting to C++17 when building hipSPARSE from source. Previously hipSPARSE was using C++14 by default.
+* Implementation of hipsparseSpMV is now using rocsparse_v2_spmv since rocsparse_spmv is deprecated.
```

Comment:
I am not sure we need to include this in the changelog as this is purely an implementation detail that does not affect the public API. In the past we have never noted these kinds of changes in the changelog I don't think.

Reply:
Yeah, I'd leave that out as it doesnt impact anyone.